website/content/en/docs/launch-templates.md

```diff
@@ -0,0 +1,207 @@
+---
+title: "Launch Templates and Custom Images"
+linkTitle: "Launch Templates"
+weight: 80
+---
+
+## Introduction
+
+Karpenter follows existing AWS patterns for customizing the base image of
+instances. More specifically, Karpenter uses [EC2 launch templates](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html). Launch
+templates may specify many values. The pivotal value is the base image (AMI).
+Launch templates further specify many parameters related to networking,

```

Comment:
It's important that we note that the majority of launch template parameters *shouldn't* be specified, since they will be overridden by Karpenter.